receiver/flinkmetricsreceiver/process.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func (s *flinkmetricsScraper) processJobmanagerMetrics(now pcommon.Timestamp, jobmanagerMetrics *models.JobmanagerMetrics) {
+	if jobmanagerMetrics == nil {
```

Comment:
Any idea what caused this to start failing now?

Reply:
I suspect there was a change that was introduced but hasn't been guarded against in the lifecycle tests.